recipes/unitig-caller/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 
 source:
   url: https://github.com/bacpop/{{ name|lower }}/archive/v{{ version }}.tar.gz
-  sha256: 61ff47df9c5339bd4cd7f6c4c7073ccb82730b645388e07369730c7c39309f33
+  sha256: 9059db7042c9b6ae240974fc474bd51d092d1f8795009a4e9e969abfc293c007
```

Comment:
There's a reason Git makes it difficult to change tags that have already been published, you know… :smile: